lib/bulk/common.js

```diff
@@ -55,60 +55,69 @@ class Batch {
 }
 
 /**
- * Create a new BulkWriteResult instance (INTERNAL TYPE, do not instantiate directly)
- *
- * @class
- * @return {BulkWriteResult} a BulkWriteResult instance
+ * @classdesc
+ * The result of a bulk write.
  */
 class BulkWriteResult {
+  /**
+   * Create a new BulkWriteResult instance (INTERNAL TYPE, do not instantiate directly)
+   *
+   * @class
+   */
   constructor(bulkResult) {
     this.result = bulkResult;
   }
 
   /**
-   * @return {boolean} ok Did bulk operation correctly execute
+   * Is true if the bulk operation correctly execute
```

Comment:
Might be worded better as `Returns true if the bulk operation correctly executes`